tests/integration/test_projectconfigs.py

```diff
@@ -269,24 +269,8 @@ def test_unparsable_project_config(buffer_config, mini_sentry, relay):
     packed, signature = SecretKey.parse(relay.secret_key).pack(body)
 
     # This request should return invalid project state and also send the error to Sentry.
-    data = get_response(relay, packed, signature)
-    assert {
-        "configs": {
-            public_key: {
-                "projectId": None,
-                "lastChange": None,
-                "disabled": True,
-                "publicKeys": [],
-                "slug": None,
-                "config": {
-                    "allowedDomains": ["*"],
-                    "trustedRelays": [],
-                    "piiConfig": None,
-                },
-                "organizationId": None,
-            }
-        }
-    } == data
+    data = request_config(relay, packed, signature, version="3").json()
+    assert data == {"configs": {}, "pending": [public_key]}
```

Comment:
Relay was already configured to buffer envelopes for unparsable projects, but downstream relays would get them as `disabled` and drop data anyway. This is now fixed.